store.go

```diff
@@ -26,12 +26,20 @@ var (
 	ErrTraceNotFound = errors.New("trace not found")
 )
 
+type TracesOpts struct {
+	Timespan Timespan
+}
+
+func DefaultTracesOpts() TracesOpts {
+	return TracesOpts{}
+}
```

Comment:
Should have docstrings on these. (for `TracesOpts`, `TracesOpts.Timespan`, and `DefaultTracesOpts`)